libs/sdk-model/src/execution/executionDefinition/index.ts

```diff
@@ -29,6 +29,18 @@ export interface IPostProcessing {
     readonly dateFormat?: string;
 }
 
+/**
+ * Contains any configiration that should be part of execution
+ *
+ * @public
+ */
+export interface IExecutionConfig {
+    /**
+     * Data sampling is only available for tiger with Vertica
```

Comment:
replace `for tiger with Vertica` with `in Tiger for specific databases` (we will support sampling for more DBs in the future)